Hangman-Game/README.md

```diff
@@ -1,12 +1,6 @@
-# Hangman Game
+# Py-game-HangMan
 
-Create a List of secret words and randomly pick a word. Now represent each word as _ and give user chances to guess the word if the user guesses the word right then replace _ with the word.
- 
-## Setup Instruction
+A simple Hangman game using Python
```

Comment:
Include requirements of the program in this readme file if you're not uploading any separate requirements.txt file